formatTest/typeCheckedTests/expected_output/arityConversion.re

```diff
@@ -16,11 +16,11 @@ module Test = {
     | Or (int, int);
 };
 
-Test.And (1, 2);
+let _ = Test.And (1, 2);
```

Comment:
So yes, we can distinguish that. However, printing with a 4.02 pretty-printer will reintroduce the loss I think. I haven't investigated yet.